wikibaseintegrator/models/claims.py

```diff
@@ -26,14 +27,12 @@ def get(self, property: str) -> List[Claim]:
         return self.claims[property]
 
     def remove(self, property: str = None) -> None:
+        """We remove all claims with the specified property
+
+        :param property: Property of claims to be removed"""
         if property in self.claims:
-            for prop in self.claims[property]:
-                if prop.id:
-                    prop.remove()
-                else:
-                    self.claims[property].remove(prop)
-            if len(self.claims[property]) == 0:
-                del self.claims[property]
```

Comment:
Looking good to me but I wonder why i make something so complex here before 🤔

Reply:
I'm guessing that you mark the claims to be removed but keep them so you know later if write is needed?

Reply:
After reviewing your code, I don't think it worked as intended.
The goal of my complex was the add a differences between already existing property (on the wikibase instances) and locally added property (not wrote to the instances) :
- If the property exist on the instance, it have an "id", so we mark the "id" as "removed"
- If the property doesn't exist (yet) on the instance, no "id", we can safely remove the property from the object.